src/main/java/org/folio/service/pieces/flows/strategies/ProcessInventoryMixedStrategy.java

```diff
@@ -59,7 +62,10 @@ private List<Future<JsonObject>> updateMixedHolding(CompositePoLine compPOL, Inv
     List<Future<JsonObject>> itemsPerHolding = new ArrayList<>();
     compPOL.getLocations().forEach(location -> itemsPerHolding.add(
       findHoldingsId(compPOL, location, restClient, requestContext)
-        .compose(aVoid -> inventoryManager.getOrCreateHoldingsJsonRecord(compPOL.getEresource(), compPOL.getInstanceId(), location, requestContext)
+        .compose(aVoid -> consortiumConfigurationService.getConsortiumConfiguration(requestContext))
```

Comment:
this common code to get consortium configuration and clone request context also can be located to utils

Reply:
I think we can move it only to separate service or leave it as it was. Because utils usually have static methods and don't need to refer to beans